blade-graphics/src/util.rs

```diff
@@ -74,6 +74,8 @@ impl super::TextureFormat {
             Self::Rg32Uint => uncompressed(8),
             Self::Rgba32Uint => uncompressed(16),
             Self::Depth32Float => uncompressed(4),
+            Self::Depth32FloatStencil8Uint => uncompressed(5),
```

Comment:
both depth and stencil are not necessarily stored together, and unlikely really occupy 5 bytes (more like 8 in this case)
let's leave a comment here that this value isn't really applicable. We should probably just set it to 0, even, to avoid confusion

Reply:
will add comment in next PR